### DISCUSSION TITLE
Computing code coverage in Handlebars templates

### DISCUSSION THREAD

**Discussion by derekstavis** [handlebars-lang/handlebars.js#1780] [Q&A]
Hey team!!

I'm wondering if there's a way to test branch code coverage on Handlebars templates, either natively or by hooking into the compiler somehow.

Thanks!

**Comment by Conormc98** [comment]
Hi, has any solution to this been raised? Currently trying to do this same thing myself.

**Comment by jaylinski** [comment]
I created an example on how to get code-coverage from handlebars-templates with Jest:

https://github.com/jaylinski/handlebars-coverage

**Reply by tniezurawski** [reply]
That's interesting! I'll definitely take a look into this. I'm wondering how we can add it to [ember-cli-code-coverage](https://github.com/kategengler/ember-cli-code-coverage) so it can go through `.hbs` files as well 🤔
